style.css

```diff
@@ -0,0 +1,708 @@
+  justify-content: center;
+  padding-top: 10px;
+}
+
+.magazine-section-header h4 {
+  padding-left: 5px;
+}
+
+.magazine-section-header-img {
+  width: 30px;
+  height: 30px;
+}
+
+.magazine-section h4 {
+  padding: 7px 0 15px 5px;
+  font-size: 19px;
+  line-height: 17px;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: 900;
+}
+
+.issue-top-container {
+  position: relative;
+}
+
+.magazine-section {
+  position: relative;
+}
+
+.main-issue {
+  position: relative;
+}
+
+.main-issue img {
+  width: 99.9%;
+}
+
+.issue-span {
+  position: absolute;
+  bottom: 0;
+  padding: 5px;
+  background-color: white;
+  color: #f61602;
+  font-weight: 700;
+}
+
+.issues-lower-link {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  background-color: #f4f5f6;
+  padding: 3px 0 5px 0;
+}
+
+.other-feature {
+  padding-bottom: 20px;
+}
+
+.features-img {
+  width: 99.9%;
+}
+
+.feature-tags {
+  display: flex;
+  padding: 7px 3px 7px 0;
+}
+
+.full-tag {
+  color: white;
+  background-color: #f61602;
+}
+
+.hollow-tag {
+  color: #f61602;
+  background-color: white;
+  padding-left: 5px;
+}
+
+.feature-link {
+  font-size: 16px;
+  line-height: 16px;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.subscribe-header h4 {
+  font-size: 16px;
+  padding-left: 5px;
+  font-family: "Roboto Condensed", sans-serif;
+  color: #f72210;
+  font-weight: 800;
+}
+
+.cont {
+  position: relative;
+  background-color: #f4f5f6;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+}
+
+.cont img {
+  max-width: 100%;
+}
+
+.btn-1 {
+  height: 40px;
+  width: 130px;
+  align-self: center;
+  margin: 10px;
+  font-weight: 800;
+}
+
+.latest-section img {
+  width: 100%;
+  height: 170px;
+}
+
+.news-center {
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  text-align: center;
+}
+
+.news-center h3 {
+  font-weight: 700;
+  padding: 20px;
+}
+
+.news-center p {
+  padding: 10px;
+  max-width: 80%;
+  font-size: 15px;
+}
+
+.languages {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 5px;
+}
+
+.contact-us {
+  padding: 5px;
+}
+
+.icons .fa {
+  color: rgba(80, 79, 79, 0.925);
+  font-size: 25px;
+  padding: 5px;
+}
+
+@media only screen and (min-width: 768px) {
+  .main-lower-nav {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: nowrap;
+  }
+
+  .myturn-section {
+    display: flex;
+    flex-flow: column;
+  }
+
+  .divider {
+    display: none;
+  }
+
+  .subscribe-img {
+    width: 250px;
+  }
+
+  .subscribe-section a,
+  .subscribe-section p {
+    text-align: left;
+  }
+
+  .medical {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 15px;
+  }
+
+  .culture {
+    display: flex;
+    column-gap: 15px;
+  }
+
+  .culture-story {
+    max-width: 30%;
+  }
+
+  .story-lg {
+    display: block;
+  }
+
+  .story-sm {
+    display: none;
+  }
+
+  .email-form {
+    display: none;
+  }
```

Comment:
- [ ] Keep code DRY, Don't Repeat Yourself. Find the other patterns and reuse the CSS rules or create CSS blocks with that specific rule. Example:

```css
# Example 1
.story-sm,
.email-form {
  display: none;
}

# Example 2
.d-none {
  display: none;
}

# Example 3

# Use display none bootstrap class instead of declaring a CSS class.
```